cypress/integration/v2/blocks/pricing-box.spec.js

```diff
@@ -63,6 +65,34 @@ function switchDesign() {
 	] ) )
 }
 
+function typeContent() {
+	it( 'should allow typing in the block', () => {
+		cy.setupWP()
+		cy.newPage()
+		cy.addBlock( 'ugb/pricing-box' ).as( 'pricingBoxBlock' )
+		registerBlockSnapshots( 'pricingBoxBlock' )
+
+		cy.openInspector( 'ugb/pricing-box', 'Style' )
+		cy.collapse( 'General' )
+		cy.adjust( 'Columns', 1 )
+
+		cy.typeBlock( 'ugb/pricing-box', '.ugb-pricing-box__title', 'Hello World! 1234' )
+			.assertBlockContent( '.ugb-pricing-box__title', 'Hello World! 1234' )
+		cy.typeBlock( 'ugb/pricing-box', '.ugb-pricing-box__price-prefix', 'P' )
+			.assertBlockContent( '.ugb-pricing-box__price-prefix', 'P' )
+		cy.typeBlock( 'ugb/pricing-box', '.ugb-pricing-box__price', '1234' )
+			.assertBlockContent( '.ugb-pricing-box__price', '1234' )
+		cy.typeBlock( 'ugb/pricing-box', '.ugb-pricing-box__price-suffix', '1234' )
+			.assertBlockContent( '.ugb-pricing-box__price-suffix', '1234' )
+		cy.typeBlock( 'ugb/pricing-box', '.ugb-pricing-box__subprice', '1234' )
+			.assertBlockContent( '.ugb-pricing-box__subprice', '1234' )
+		cy.typeBlock( 'ugb/pricing-box', '.ugb-button--inner', 'Hello World! 1234' )
+			.assertBlockContent( '.ugb-button--inner', 'Hello World! 1234' )
+		cy.typeBlock( 'ugb/pricing-box', '.ugb-pricing-box__description', 'Hello World! 1234' )
+			.assertBlockContent( '.ugb-pricing-box__description', 'Hello World! 1234' )
```

Comment:
Try using different text for different assertions so that you're sure that what's typed didn't affect other fields

Reply:
Okay thank you! I'll vary these as well